explore_utils.py

```diff
@@ -20,6 +20,48 @@ def find_most_visit(dataset):
     return max_visit
 
 
+def find_fraction_of_customers_with_non_zero_revenue(dataset):
+    """Find the fraction of customers in the google dataset with non-zero revenue.(only in train set)
+    
+    args:
+        dataset (Dataset): the google analytics dataset.
+        
+    returns:
+            The fraction of customers with non-zero revenue.
+    """
+    
+    
+    train_df = dataset.train
+    train_df['revenue'] = train_df['totals.transactionRevenue'].astype(float)
```

Comment:
This adds a new column to `train_df` which will be reflected back in `explore.py`. please see the comment I posted in PR #49 for info on how to fix.